hangul/hangulkeyboard.c

```diff
@@ -815,6 +823,53 @@ hangul_keyboard_list_load_dir(const char* path)
 
     globfree(&result);
     free(pattern);
+#elif _WIN32
+    WIN32_FIND_DATA FindFileData;
```

Comment:
```suggestion
    WIN32_FIND_DATA findFileData;
```
로컬 변수 이름은 소문자로 시작해주시면 좋겠습니다.